mta/tests/migration/controls/test_stakeholder.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_email, generate_string
+
+
+@pytest.mark.stakeholder
+def test_stakeholder_crud(stakeholder, get_api, delete_api, update_api):
+    # get stakeholder_from_db
+    stakeholder_from_db = get_api.stakeholders_id_get(stakeholder.id)
+
+    # Assertion checking the created stakeholders name
+    check.equal(stakeholder_from_db.name, stakeholder.name)
+
+    # Edit the stakeholder name and description
+    stakeholder.name = generate_string(start="update name")
+    stakeholder.email = generate_email()
+    update_api.stakeholders_id_put(stakeholder.id, stakeholder.to_dict())
+    update_stakeholder_from_db = get_api.stakeholders_id_get(stakeholder.id)
+    # Assertion checking for the updated stakeholders name
+    check.equal(update_stakeholder_from_db.name, stakeholder.name)
+
+    # Delete the Stakeholder
+    delete_api.stakeholders_id_delete(str(stakeholder.id))
+    delete_api.stakeholdergroups_id_delete(str(stakeholder.id))
```

Comment:
Please use a teardown fixture to delete MTA objects

Reply:
Not needed as this is the only test for stakeholder group.

Reply:
@sshveta @kpunwatk It is not about what has been tested but about the stakeholder fixture that is in use in 2 places: BS test and stakeholder test. In both, there is a **stakeholder delete**, which means that if both tests are running - the second one will fail because the stakeholder is deleted already.
See: https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-api-tests-runner/20/console
We must use a teardown fixture for MTA objects created to avoid such a failure. Thanks